test/test-hash.expected

```diff
@@ -0,0 +1,20 @@
+
+<DESCRIBE::>string hash
+
+<IT::>short strings
+
+<PASSED::>Test Passed
+
+<PASSED::>Test Passed
+
+<COMPLETEDIN::>0.098 ms
+
+<IT::>failing compares
+
+<FAILED::>Expected 431880812 , got 390772129 
+
+<FAILED::>Expected 143064460 , got 431880812 
```

Comment:
> I think the benefit of being able to test string equality currently outweighs that disadvantage.

I understand this is difficult (from your comment https://github.com/Codewars/codewars-runner-cli/issues/625#issuecomment-408760815), but this will be extremely frustrating for users.

You can't see expected or actual. I guess the authors can print those above it?

Reply:
Yes, in the tests the author of the Kata can create his own helper function to print strings. 

In general, the difficult problem to solve in Forth is that it's data stack is untyped, and that words/functions return multiple values. In the case of strings, that are two values; a char address and a count. This could be solved, but requires substantially more work than this simple solution of hashing strings into a single value on the datastack.